flink-end-to-end-tests/pom.xml

```diff
@@ -90,6 +162,66 @@ under the License.
 					<skip>true</skip>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.codehaus.gmavenplus</groupId>
+				<artifactId>gmavenplus-plugin</artifactId>
+				<version>1.8.1</version>
+				<dependencies>
+					<dependency>
+						<groupId>org.codehaus.groovy</groupId>
+						<artifactId>groovy-all</artifactId>
+						<version>2.5.9</version>
+						<scope>runtime</scope>
+						<type>pom</type>
+					</dependency>
+				</dependencies>
+				<executions>
+					<execution>
+						<id>merge-categories</id>
+						<phase>initialize</phase>
+						<goals>
+							<goal>execute</goal>
+						</goals>
+						<configuration>
+							<scripts>
+								<script>
+									<![CDATA[
+									def categorySearcher = { String propertyPrefix ->
+										project.properties.entrySet().stream()
+											.filter { p -> p.getKey().startsWith(propertyPrefix) }
+											.filter { p -> p.getValue().toBoolean() }
+											.map { p -> p.getKey() }
+											.map { p -> p.substring(propertyPrefix.size(), p.size()) }
+											.toList()
+									}
+
+									def includes = categorySearcher('e2e.include.')
+									// allow overrides from the command-line
+									if (!project.properties.includeE2E.isEmpty()) {
+										includes.add(project.properties.includeE2E)
+									}
+									if (includes.size() > 0) {
+										project.properties.includeE2E = includes.join(',')
+									} else {
+										project.properties.includeE2E = project.properties.'includeE2E.default'
+									}
+
+									def excludes = categorySearcher('e2e.exclude.')
+									// allow overrides from the command-line
+									if (!project.properties.excludeE2E.isEmpty()) {
+										excludes.add(project.properties.excludeE2E)
+									}
+									project.properties.excludeE2E = excludes.join(',')
+
+									println "includes: " + project.properties.includeE2E
+									println "excludes: " + project.properties.excludeE2E
```

Comment:
Do we want to keep these `printlns`?

Reply:
I'm not sure. I like that you can tell at a glance whether something is off, which will be useful on Travis.

Reply:
maybe keep them for a bit until the transition to azure is done and we are confident it works as it should.